txengine-deploy/src/env-config.js

```diff
@@ -33,6 +33,7 @@ const defaultEnvironment = (projectId, tenantName, countryCode) => ({
   PGPASSWORD: `sm://${projectId}/${tenantName}_${getConditionalCountryCodeString(countryCode)}postgresql_master_password`,
   SERVICE_PROJECT_ID: projectId,
   SERVICE_ENVIRONMENT: projectId.includes('-staging-') ? 'staging' : 'prod',
+  LAUNCH_DARKLY_ACCESS_KEY: `sm://${projectId}/launchdarkly-sdk-key`,
```

Comment:
Just to be 100% sure, this `launchdarkly-sdk-key` is the key that is automatically set with the TF automation?

Reply:
Yes, that's correct!